packages/admin/admin/package.json

```diff
@@ -64,6 +66,12 @@
         "react-router-dom": "^5.1.2"
     },
     "peerDependenciesMeta": {
+        "@mui/x-data-grid-premium": {
+            "optional": true
+        },
+        "@mui/x-data-grid-pro": {
+            "optional": true
+        },
```

Comment:
Shouldn't `@mui/x-data-grid` be marked as optional as well? Otherwise, both the normal and the premium version will be installed, as NPM installs peer dependencies automatically since v7. For the types we could add the normal version as dev dependency

Reply:
we have a runtime dependency on it: https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/src/dataGrid/usePersistentColumnState.tsx#L1